modules/admin_manual/pages/enterprise/external_storage/sharepoint-integration_configuration.adoc

```diff
@@ -19,8 +19,8 @@ properly accessed as per SharePoint rules.
 
 The plugin uses the Simple Object Access Protocol (SOAP) and WebDAV for
 the uploads and downloads to talk to SharePoint servers. Your ownCloud
-server must have `php-soap` or `php5-soap` installed. Linux packages and
-ownCloud appliances will install `php5-soap` as a required dependency.
+server must have `php-seclib` and `php-soap` or `php7-soap` installed; and enabled soap as a php module.
```

Comment:
not sure about php-seclib.
The actual package name is php-phpseclib but then it still does not seem to work. 
More investigation needed.